go/ct/spc/specification.go

```diff
@@ -2196,6 +2198,9 @@ func callEffect(s *st.State, addrAccessCost vm.Gas, op OpCode) {
 		recipient = s.CallContext.AccountAddress
 		codeAddress = target.Bytes20be()
 		value = s.CallContext.Value
+	} else if op == CALLCODE {
+		kind = vm.CallCode
+		sender = s.CallContext.AccountAddress
```

Comment:
I would expect two more lines here, please check whether this is how it is supposed to work:
```
recipient := s.CallContext.AccountAddress
codeAddress := target.Bytes20be()
```

For reference, maybe check the evmzero implementation [here](https://github.com/ethereum/evmone/blob/648313893b42e5f1e0c1453783fa208e8c6a9d88/lib/evmone/instructions_calls.cpp#L54-L64).